dbt/include/snowflake/macros/relations/dynamic_table/replace.sql

```diff
@@ -1,18 +1,84 @@
 {% macro snowflake__get_replace_dynamic_table_sql(relation, sql) -%}
+{#-
```

Comment:
These doc blocks could be helpful. They also add a ton of lines to the file and they're not as easy to read and mentally separate from the functional code as they would be in python. I could take them or leave them. I do want to keep the urls referenced below though, since these macros mirror those docs very closely.

Reply:
I like the doc blocks but I agree they are harder to read unfortunately. Worth the tradeoff of including them for now imo.

One other thing is to have them in Jinja {# #} then prefacing each line with -- to make them read as SQL comments, which helps.

Reply:
> One other thing is to have them in Jinja {# #} then prefacing each line with -- to make them read as SQL comments, which helps.

Good idea. I'll update it. I thought about how docstrings are easier to read in python because the IDE formats it differently. I didn't think to do that though.